### PR TITLE
draw_line, draw_v_line, draw_sprite_or, draw_sprite_and, draw_sprite_xor, draw_char, draw_short, draw_byte, draw_long

### PR DESCRIPTION
**Pull request by MaxLeiter**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,9 +94,9 @@ clean:
 package: all
 	kpack libc-$(VERSION).pkg $(ROOT)
 
+
 install: package
-	kpack -e -s libc-$(VERSION).pkg $(PREFIX)
-	
+	kpack -e -s libc-$(VERSION).pkg $(PREFIX)
```
Comment (SirCmpwn) [inline Makefile]: Why this change?

File: src/knightos/display.c [hunk 2]
```diff
@@ -126,3 +267,11 @@ void draw_short(SCREEN *screen, unsigned char x, unsigned char y, unsigned short
...
 	screen; x; y; value;
 }
+
+/** TODO: make this work, ACIX **/
+void draw_long(SCREEN *screen, unsigned char x, unsigned char y, unsigned long value) {
+	__asm 
+		PCALL(DRAWDECACIX)
+	__endasm;
+	screen; x; y; value;
+}
```
Comment (SirCmpwn) [inline src/knightos/display.c]: If it doesn't work, don't include it in the PR.

**Comment by SirCmpwn** [comment]
Since this is a breaking API change, can you please send a corresponding PR to pong?

**Comment by MaxLeiter** [comment]
Working on that now; www.github.com/maxleiter/pong is half-updated atm

**Comment by MaxLeiter** [comment]
^ @SirCmpwn
